.ci/test-spelling-unknown-words.sh

```diff
@@ -55,7 +55,8 @@ echo "Run w"
 
 printDetails() {
   echo ''
-  echo 'If you are ok with the output of this run, you will need to'
+  echo 'If you are ok with the output of this run, you will need \
+to run the following entire multiline command:'
```

Comment:
I would do this in two separate echo commands, no reason on line wrap here.

Reply:
`echo 'If you are ok with the output of this run,'`
Other text by separate echo command.

Reply:
Done